driver/src/main/java/com/impossibl/postgres/api/jdbc/PGSQLExceptionInfo.java

```diff
@@ -55,6 +55,10 @@
 
   String getConstraint();
 
-  void setConstraint(String constraint);
```

Comment:
Unnecessarily renamed?

Reply:
Hm, that was surely not intended, fixed now.